rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-download-from-google-drive.adoc

```diff
@@ -0,0 +1,168 @@
+= Download from Google Drive 
+
+Download or Export files from your Google Drive. 
+
+You can export Google Docs, Sheets, and Slides into a selection of supported formats including PDF.
+
+== Before You Begin 
+
+Before downloading a file, ensure you: 
```

Comment:
You could make these present tense/imperative.

Before downloading a file:
- Configure
- Obtain